src/server/cray/cfs/api/controllers/components.py

```diff
@@ -284,7 +499,18 @@ def patch_component(component_id):
 
 
 @dbutils.redis_error_handler
-def delete_component(component_id):
+def delete_component_v2(component_id):
+    """Used by the DELETE /components/{component_id} API operation"""
+    LOGGER.debug("DELETE /components/id invoked delete_component")
+    if component_id not in DB:
+        return connexion.problem(
+            status=404, title="Component could not found.",
+            detail="Component {} could not be found".format(component_id))
+    return DB.delete(component_id), 204
+
+
+@dbutils.redis_error_handler
+def delete_component_v3(component_id):
     """Used by the DELETE /components/{component_id} API operation"""
     LOGGER.debug("DELETE /components/id invoked delete_component")
     if component_id not in DB:
```

Comment:
The "could not found" typo is also a couple lines down, just beyond where github will let me make a suggestion